ng-simple-webrtc.js

```diff
@@ -214,32 +227,23 @@
                 maxWidth: minWidth
               };
             }
+            return webrtcOptions;
+          }
 
-            function displayVideoResolution() {
```

Comment:
Feel free to put it back.

When we started using 'onloadedmetadeta', there was a bunch of code in
show-whiteboard-simple-webrtc.js that could be removed.
I was pruned a bunch of code -- I could easily have overdone it.

 Thanks!

-- Samson

On Sat, Sep 12, 2015 at 9:11 AM, Gleb Bahmutov <notifications@github.com>
wrote:

> In ng-simple-webrtc.js
> <https://github.com/bahmutov/ng-simple-webrtc/pull/12#discussion_r39337196>
> :
>
> >
> > -            function displayVideoResolution() {
>
> why did you remove this method?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bahmutov/ng-simple-webrtc/pull/12/files#r39337196>.
>